XU-TechnicalPaper.md

```diff
@@ -102,17 +102,60 @@ Payment channel atomic swap technology is fairly new and was just recently succe
 Since Ethereum includes all ERC20 tokens, Exchange Union supports the vast majority of today’s token market. [Other important chains](https://storeofvalue.github.io/posts/what-is-qtum-without-the-bullshit/) are expected to start developing Lightning/Raiden-style payment channel technology within the next months. Options how to support some in the short term, e.g. via [ERC20 wrapper](https://weth.io/) technology, can be explored.
 
 
-3.3. Decentralized Orderbooks
+3.3. Decentralized Order Books (DOB)
 --------------------------
-Decentralized orderbook propagation is a key task for XU and a problem on existing decentralized exchanges where, to the best of our knowledge, this is mostly realized in a centralized way e.g. [1](https://github.com/etherdelta/etherdelta.github.io/blob/master/docs/API.md) or [2](https://github.com/0xProject/standard-relayer-api/blob/master/http/v0.md). We believe, orderbook propagation has to occur in a decentralized manner to sustain the overall robustness and censorship resistance of the system. Ideally the propagation would happen directly within a default XUC (raiden) payment channel as payload of a microtransaction. This would have the advantage, that order relay fees are paid at the same time and only a successfully forwarded order would let the intermediate node redeem its earned fee. Also a default XUC payment channel is required anyways for our incentive system.
+The DOB is a key task for XU and an unresolved problem on existing decentralized exchanges where, to the best of our knowledge, the order book part is realized in a centralized way e.g. [1](https://github.com/etherdelta/etherdelta.github.io/blob/master/docs/API.md) or [2](https://github.com/0xProject/standard-relayer-api/blob/master/http/v0.md). We believe, order book propagation and matching has to be decentralized to the best degree possible to sustain the overall robustness and censorship resistance of the network.
 
+DOB's mainly face the following issues:
 
+**Latency**
+The speed of order updates is crucial to keep order information as accurate as possible and also minimize noise ('useless' traffic) on the network resulting from outdated order information. For example nodes requesting to fill orders, which are already filled. Since we target to on-board high volume exchanges, this network load is potentially huge, as well as the impact of resulting delays caused by these race conditions.
```

Comment:
Removed race conditions, not sure if it's really fitting here anyways. Revised the 'delay' sentence I meant the delay until an order is filled, CAUSED by order constantly failing because outdated info because of latency. 🗡